tools/pipelines/test-real-service.yml

```diff
@@ -168,3 +168,40 @@ stages:
           login__microsoft__secret: $(login-microsoft-secret)
           login__odsp__test__tenants: $(automation-e2e-login-odsp-test-tenants)
           FLUID_TEST_LOGGER_PKG_PATH: ${{ variables.testWorkspace }}/node_modules/@ff-internal/aria-logger # Contains getTestLogger impl to inject
+  #Capture per-pipeline stage results
+  - stage: runAfterAll
+    displayName: Retrieve and upload E2E per-pipeline stage stats to Kusto
+    condition: always()
+    dependsOn:
+      - e2e_local_server
+      - e2e_tinylicious
+      - e2e_routerlicious
+      - e2e_frs
+      - e2e_odsp
+
+    jobs:
+      - template: templates/include-test-perf-benchmarks.yml
```

Comment:
Not to be addressed here but I just noticed that we're reusing this template (whose name references the perf benchmarks pipeline) here and in the stress tests pipeline for the stage to report stats... probably not ideally named now, and it gives the name "Run tests" to the job, which is also not accurate for what's happening. I'll create an ADO task to track some refactoring in this area.